packages/esm-patient-labs-app/src/lab-orders/add-lab-order/useTestTypes.test.ts

```diff
@@ -0,0 +1,97 @@
+import { renderHook, waitFor } from '@testing-library/react';
+import { useTestTypes } from './useTestTypes';
+import { defineConfigSchema, getDefaultsFromConfigSchema, openmrsFetch, useConfig } from '@openmrs/esm-framework';
+import { ConfigObject, configSchema } from '../../config-schema';
+import useSWRImmutable from 'swr/immutable';
+import { useEffect, useRef, useState } from 'react';
+
+jest.mock('swr/immutable');
+
+const mockOpenrsFetch = openmrsFetch as jest.Mock;
+const mockUseConfig = useConfig as jest.Mock;
```

Comment:
Btw, `jest.mocked(moduleName)` gives you access to the source's original type annotations in your mock object for free.

Reply:
Nice, will try that out